debian/rpi-eeprom.postrm

```diff
@@ -20,9 +20,6 @@ set -e
 
 
 case "$1" in
-    purge)
-        rm -rf /var/lib/raspberrypi/bootloader/backup/
```

Comment:
Could you elaborate on this change a bit? It still seems to create that directory here https://github.com/raspberrypi/rpi-eeprom/blob/0125155f9da31596252bbff4431d1bb7d8259933/rpi-eeprom-update#L112

Reply:
> Could you elaborate on this change a bit? It still seems to create that directory here
> 
> https://github.com/raspberrypi/rpi-eeprom/blob/0125155f9da31596252bbff4431d1bb7d8259933/rpi-eeprom-update#L112

I've probably done this the wrong way - the packaging change depends on #249, which corrects ${FIRMWARE_BACKUP_DIR} to be ${FIRMWARE_DIR}/backup, which ends up as `/lib/firmware/raspberrypi/firmware/backup`.

Reply:
Also see my comment above.

Reply:
I meant why was the removed from purge, but you've answered that in the edit.

I think `purge` implies that you want it all gone. When `piuparts` runs, it checks that there is no cruft left over at all after a purge and this would fail that check. If the user wants to keep backups, they shouldn't purge them, right?